src/node_options.cc

```diff
@@ -86,6 +86,11 @@ void PerIsolateOptions::CheckOptions(std::vector<std::string>* errors) {
     return;
   }
 
+  if (per_process::cli_options->report_on_fatalerror) {
```

Comment:
I think we should omit this, as https://github.com/nodejs/node/pull/32242 makes it unnecessary, as previously noted.

Reply:
@cjihrig, removed this block. PTAL.